src/test/java/com/bobocode/svydovets/beans/BeanScanner.java

```diff
@@ -0,0 +1,8 @@
+package com.bobocode.svydovets.beans;
```

Comment:
What do you think about moving it to a specific package? For example, scanner or discovery, etc.

Reply:
Done